test/sparrow_test.exs

```diff
@@ -33,6 +33,7 @@ defmodule SparrowTest do
   end
 
   test "Sparrow starts correctly", context do
+    {:ok, pid} = start_supervised(Sparrow.PoolsWarden)
```

Comment:
This looks lik a context for me. Can it be extracted there? See: https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-context